packs/feat-effects/effect-plague-rat-curse.json

```diff
@@ -0,0 +1,36 @@
+{
+    "_id": "0dJV8I7w4RIabM9j",
+    "img": "systems/pf2e/icons/spells/ravening-maw.webp",
+    "name": "Effect: Plague Rat Curse",
+    "system": {
+        "description": {
+            "value": "<p>Your bites carry a cursed infection. When you hit and deal damage to a creature with a jaws Strike while in hybrid or rat shape, the creature is cursed until the start of your next turn. Whenever they regain Hit Points during that time, they must attempt a @Check[fortitude|dc:resolve(@item.origin.system.attributes.classOrSpellDC.value)|rollerRole:target|showDC:all] save against your class DC or spell DC, whichever is higher.</p><hr /><p><span style=\"font-family:'Inter var', sans-serif\"><strong>Success</strong> The creature regains Hit Points normally.</span></p>\n<p><strong>Failure</strong> The creature regains half the normal Hit Points.</p>\n<p><strong>Critical Failure</strong> The creature regains no Hit Points.</p>"
```

Comment:
Remove `<span style=\"font-family:'Inter var', sans-serif\">`
Formatting doesn't get applied to description text.

Reply:
I missed that, but it's obsolete now because the effect no longer exists.